Dockerfile

```diff
@@ -19,6 +19,8 @@ COPY ./ /kb/module
 RUN mkdir -p /kb/module/work
 RUN chmod -R a+rw /kb/module
 
+RUN curl -s https://raw.githubusercontent.com/ModelSEED/PlantSEED/master/Data/PlantSEED_v2/Flat_Files/Supporting_Data_S3_Signature_Kmers.txt > /kb/module/data/functions_kmers.txt
```

Comment:
How big is this file? If it's really big, using the reference data format is more efficient because it loads the data on a shared drive rather than pulling it down to the individual workers.

Reply:
If it's that small stashing it in /data is probably fine